Sources/StreamChatUI/StreamSwiftyMarkdown/SwiftyMarkdown/SwiftyLineProcessor.swift

```diff
@@ -250,7 +250,7 @@ class SwiftyLineProcessor {
     }
     
     func processOrderedListRegex(_ text: String) -> String {
-        let regex = try? NSRegularExpression(pattern: "^[0-9]+. ", options: .caseInsensitive)
+        let regex = try? NSRegularExpression(pattern: "^[0-9]+\\. ", options: .caseInsensitive)
```

Comment:
One unit-test for capturing this issue would be beneficial. E.g. if we change our markdown parser, then this issue would also get retested.

Reply:
I've added a snapshot test. Let me know if you think it's not enough 👍

Reply:
Sounds good!